includes/wikia/models/tests/DesignSystemSharedLinksTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+class DesignSystemSharedLinksTest extends WikiaBaseTest {
+
+	/**
+	 * @dataProvider getHrefDataProvider
+	 *
+	 * @param $lang language code to fetch
+	 * @param $hrefs hrefs definition in different languages
+	 * @param $expectedResult
+	 */
+	public function testGetHref( $lang, $hrefs, $expectedResult ) {
+		DesignSystemSharedLinks::getInstance()->setHrefs( $hrefs );
+
+		$result = DesignSystemSharedLinks::getInstance()->getHref( 'create-new-wiki', $lang );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "create-new-wiki" 10 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)